include/proxy-wasm/compat.h

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
Is this repo using the Google CLA? Or DCO? If using the Google CLA that's a serious problem that we will need to discuss ASAP. I'm also not thrilled w/ the Google copyright here. If the plan here is to move this repo into community ownership can we please strip this header (since it's generally understood it's not needed on every file). If we don't want to trip the header can we please either use "Envoy Proxy Authors" or "Istio Authors" (less good IMO).

Reply:
I see https://github.com/proxy-wasm/proxy-wasm-cpp-host/blob/master/CONTRIBUTING.md so this is using Google CLA. This is acceptable from my perspective. Can we please move this to DCO and remove the Google CLA? Thank you!

Reply:
I talked to @louiscryan about this offline. I will ignore this for now and we can circle back in 1-2 weeks. Thanks!